packages/web/src/scss/components/UNSTABLE_Slider/index.html

```diff
@@ -0,0 +1,223 @@
+{{#> web/layout/plain }}
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="UNSTABLE_Slider">
+      <label for="slider-default" class="UNSTABLE_Slider__label">
+        Slider
+      </label>
+      <input
+        class="UNSTABLE_Slider__input"
+        id="slider-default"
+        max="100"
+        min="1"
+        step="1"
+        style="--slider-position: 30%"
+        type="range"
+        value="30"
+        oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+      />
+    </div>
+
+  </div>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Selected</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <!-- TODO: Resolve our approach to the selected / unselected form field states. -->
+
+  <div class="UNSTABLE_Slider UNSTABLE_Slider--selected">
+    <label for="slider-selected" class="UNSTABLE_Slider__label">
+      Slider
+    </label>
+    <input
+      class="UNSTABLE_Slider__input"
+      id="slider-selected"
+      max="100"
+      min="1"
+      step="1"
+      style="--slider-position: 30%"
+      type="range"
+      value="30"
+      oninput="this.style.setProperty('--slider-position', `${Math.round((100 * this.value) / 100)}%`);"
+    />
+  </div>
+
+  </div>
+</section>
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Required</h2>
+  <div class="docs-Stack docs-Stack--stretch">
+
+    <div class="UNSTABLE_Slider">
+      <label for="slider-required" class="UNSTABLE_Slider__label UNSTABLE_Slider__label--required">
+        Slider
+      </label>
```

Comment:
Since this is just a problem of the web package, we decided to adjust HTML formatting here and turn labels into single-line code. Prettier can still break it, but we can tackle that once it happens.